test/item.test.js

```diff
@@ -0,0 +1,18 @@
+const Item = require('../src/item')
+const assert = require('assert').strict
```

Comment:
Habe hier kurz dazu recherchiert. Es gibt Möglichkeiten, dass wir import stattdessen verweden. Allerdings müssten wir dann in der package.json was umschreiben oder mjs-files verwenden. Was meinst du dazu